content_creator.py

```diff
@@ -63,44 +62,48 @@ def convert_incident_fields_to_array():
                 f.truncate()
 
 
-def copy_dir_yml(dir_name, version_num, bundle_pre, bundle_post, bundle_test):
+def copy_yaml_post(path, out_path, yml_info):
+    dirname = os.path.dirname(path)
+    if dirname in DIR_TO_PREFIX.keys() and not os.path.basename(path).startswith('playbook-'):
+        script_obj = yml_info
+        if dirname != 'Scripts':
+            script_obj = yml_info['script']
+        with io.open(path, mode='r', encoding='utf-8') as f:
+            yml_text = f.read()
+        out_map = write_yaml_with_docker(out_path, yml_text, yml_info, script_obj)
+        if len(out_map.keys()) > 1:
+            print(" - yaml generated multiple files: {}".format(out_map.keys()))
+        return
+    # not a script or integration file. Simply copy
+    shutil.copyfile(path, out_path)
+
+
+def copy_dir_yml(dir_name, bundle_post):
     scan_files = glob.glob(os.path.join(dir_name, '*.yml'))
     post_files = 0
     for path in scan_files:
         with open(path, 'r') as f:
             yml_info = yaml.safe_load(f)
 
         ver = yml_info.get('fromversion', '0')
-        if ver == '' or is_ge_version(version_num, ver):
-            print ' - marked as post: %s (%s)' % (ver, path, )
-            shutil.copyfile(path, os.path.join(bundle_post, os.path.basename(path)))
-            shutil.copyfile(path, os.path.join(bundle_test, os.path.basename(path)))
-            post_files += 1
-        else:
-            # add the file to both bundles
-            print ' - marked as pre: %s (%s)' % (ver, path, )
-            shutil.copyfile(path, os.path.join(bundle_pre, os.path.basename(path)))
-            shutil.copyfile(path, os.path.join(bundle_post, os.path.basename(path)))
+        print ' - processing: %s (%s)' % (ver, path, )
```

Comment:
better to use format

Reply:
This whole script needs to be moved to py3 and cleaned up. We will do this as a separate initiative.